### PR TITLE
Add missing config params to reg service deployment

### PR DESCRIPTION
**Pull request by sbryzak**
This PR adds support for a few missing configuration parameters for phone verification in the registration service deployment.

Fixes https://issues.redhat.com/browse/CRT-811

### CODE REVIEW

**Review by alexeykazakov — APPROVED** [review]

**Review by tinakurian — APPROVED** [review]

File: deploy/env/e2e-tests.yaml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 registration-service:
   environment: 'e2e-tests'
   replicas: 2
+  verification:
+    enabled: 'true'
+    excluded-email-domains: 'redhat.com,acme.com'
+  twilio:
+    account-sid: ''
```
Comment (tinakurian) [inline deploy/env/e2e-tests.yaml:8]: do we need to add these here if the default is empty?

**Review by tinakurian — CHANGES_REQUESTED** [review]
i think we shouldn't add the twilio fields here if they are just set to empty? empty can be the default and we currently don't specify fields in the yamls that contain defaults because configuration will take care of that

File: deploy/env/e2e-tests.yaml (hunk 1 above)
Comment (sbryzak) [reply deploy/env/e2e-tests.yaml:8]: I didn't intend to add these changes yet.. I've commented them out until the e2e tests are running successfully.

**Pull request merged**